### ISSUE TITLE
[context menu] Add keyboard navigation and accessible text

### ISSUE THREAD

**Issue by markov00** [elastic/elastic-charts#2133]
The tooltip accessibility has been left out because the chart geometries are not accessible yet and you probably can't reach the tooltip by navigation with the keyboard. 
We should work on making geometry accessible from keyboard and subsequently add the tooltip accessibility too.

blocked by #300
